shared/actions/chat/attachment.js

```diff
@@ -93,24 +118,28 @@ function * onLoadAttachment ({payload: {conversationIDKey, messageID, loadPrevie
     'chat.1.chatUi.chatAttachmentDownloadDone',
   ])
 
-  const channelMap = ((yield call(ChatTypes.localDownloadFileAttachmentLocalRpcChannelMap, channelConfig, {param})): any)
+  try {
```

Comment:
This is because we now set the message state to `'downloading'`/`'downloading-preview'` before kicking off the RPC. We need to reset the state of the message if it fails.